### DISCUSSION TITLE
need support osx-arm64

### DISCUSSION THREAD

**Discussion by WangHHY19931001** [microsoft/vcpkg#30636] [Ideas]
due to apple new plantform change to arm64, i think osx-arm64 will been support in official

**Comment by jimwang118** [comment]
Has the osx dependent platform been changed to arm64 and has it been released on Apple's official website? Where can I find this information?

**Comment by WangHHY19931001** [comment]
in apple office web, can't buy intel new pc any more with out mac-pro

**Comment by WangHHY19931001** [comment]
> Has the osx dependent platform been changed to arm64 and has it been released on Apple's official website? Where can I find this information?

with WWDC20 apple say change macos to base arm64

**Comment by Nosenzor** [comment]
osx-arm64 needs to be integrated in officially supported triplets with full range of build test.  osx-arm64 has been there since 2020 now.
